packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -231,6 +231,9 @@ class FakeDevice extends Fake implements Device {
   @override
   Future<TargetPlatform> get targetPlatform async => _targetPlatform;
 
+  @override
+  final PlatformType platformType = PlatformType.ios;
```

Comment:
Intentional as part of this change?

Reply:
Yes. `Device.platformType` is now queried by `FlutterDevice.create()`, and this matches the value of `targetPlatform` for this device.